### ISSUE TITLE
The gene2transcripts endpoint returns empty results for some genes

### ISSUE THREAD

**Issue by beboche** [openvar/rest_variantValidator#71]
HI,
a bunch of genes return an empty transcript object in the json output.

Example of A3GALT2:
https://www608.lamp.le.ac.uk/VariantValidator/tools/gene2transcripts/A3GALT2?content-type=application%2Fjson

returns:
{
  "current_name": "alpha 1,3-galactosyltransferase 2",
  "current_symbol": "A3GALT2",
  "hgnc": "HGNC:30005",
  "previous_symbol": "A3GALT2P",
  "transcripts": []
}

HGNC says there is a MANE and a refseq transcript: NM_001080438.1, as well as the NCBI
https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:30005
https://www.ncbi.nlm.nih.gov/gene/?term=A3GALT2

I have this for those genes:
A3GALT2
ALG1L
ANKRD20A4P
C19orf48
C3orf56
CRYBG3
DNAH17
EFCAB8
ELOA3CP
FAM169B
FAM86C1P
LRP5L
OR14K1
OR1E3
OR2B3
OR52Z1
OR8S1
PANO1
POM121L2
TCP10L2

Thanks in advance,

david

**Comment by John-F-Wagstaff** [comment]
Yes NM_001080438 was marked as permanently suppressed 2009.07.27, but is now active again, several of the others fit this pattern. This silent re-appearance of revoked records is not just an issue for us, for example (at the time of writing) TCP10L2's latest NCBI transcript is also listed as not alive/removed in the CCDS report, despite having been un-revoked by the NCBI.  We hope to reactivate all of the currently valid sequences on our next VVTA release, to fix this. However some of the genes in your list are supposed to be suppressed.
For example C19orf48 
If you go from, the  gene report to the CCDS record i .e. from
https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:29667
to
https://www.ncbi.nlm.nih.gov/projects/CCDS/CcdsBrowse.cgi?REQUEST=ALLFIELDS&DATA=CCDS12803
you will see that the status is "Reviewed, withdrawal pending" and all of the current NCBI data for this record has been revoked, checking just the last record in the NCBI shows
NM_032712.3: This RefSeq was removed because it is now thought that this gene is a pseudogene.
So since we are not currently giving Ensembl transcripts for this feature then this is the expected result.

**Comment by beboche** [comment]
ok thanks for the answer. Are you planning regular VVTA updates?

**Comment by John-F-Wagstaff** [comment]
We are hopping to update at least once a year, maybe up to once every 6 months, time and funding permitting.

**Comment by beboche** [comment]
It would be nice to be able to query this endpoint using HGNC IDsn, wouldn't it be?

**Comment by i3hsInnovation** [comment]
Hi @beboche . Yes, nice idea. I'll open this as a new issue since it's a VV thing not a database thing. I'll link it here
It's Pete by the way. Just logged in as another user!!!

**Comment by i3hsInnovation** [comment]
https://github.com/openvar/variantValidator/issues/323

**Comment by i3hsInnovation** [comment]
> We are hopping to update at least once a year, maybe up to once every 6 months, time and funding permitting.
We are working on a new build now. It's taking a while because we are also setting up infrastructure, but John is all over the problem

**Comment by beboche** [comment]
> Hi @beboche . Yes, nice idea. I'll open this as a new issue since it's a VV thing not a database thing. I'll link it here It's Pete by the way. Just logged in as another user!!!

perfect!

**Comment by i3hsInnovation** [comment]
Just need a few staff now like @John-F-Wagstaff . Anyone want to buy a kidney? :P

**Comment by beboche** [comment]
Hi,
Pete told me to add this one here:

That kind of query on chrM:
https://rest.variantvalidator.org/VariantValidator/variantvalidator/hg19/M-7425-A-G/all?content-type=application/json
returns the error:
"Failed to fetch NC_001807.4 from SeqRepo (/local/seqrepo/VV_SR_2021_2/master) (Alias NC_001807.4 (namespace: None))"
